Firestore/Source/Util/FSTDispatchQueue.h

```diff
@@ -74,6 +74,13 @@ typedef NS_ENUM(NSInteger, FSTTimerID) {
  */
 - (void)verifyIsCurrentQueue;
 
+/**
+ * Declares that a dispatch queue operation is already in progress. To be used only when called
```

Comment:
nit: "Operation is already in progress" seems a bit unclear since operationInProgress will /not/ already be true. Perhaps reword to something like:

Declares that we are already executing on the correct dispatch_queue_t and would like to officially execute code on behalf of this FSTDispatchQueue.